code/modules/mob/_modifiers/modifiers_misc.dm

```diff
@@ -0,0 +1,182 @@
+
+After the modifier expires, a second modifier representing exhaustion is placed, which inflicts massive maluses and prevents
+further berserks until it expires. This generally means that someone getting caught while exhausted will be much easier to fight,
+as they will be much slower, attack slower, do less melee damage, be easier to hit, and disabling effects will affect them harder.
+
+Berserking causes the holder to feel hungrier. If they are starving, this modifier cannot be applied. Diona cannot
+be berserked, or those who are suffering from exhaustion. Non-Drone Synthetics that receive the berserk modifier will
+instead get a version that has no benefits, but will not cost nutrition or cause exhaustion. Drones cannot receive berserk, as
+they are emotionless automatrons.
+
+Berserk is a somewhat rare modifier to obtain freely (and for good reason), however here are ways to see it in action;
+- Red Slimes will berserk if they go rabid.
+- Red slime core reactions will berserk slimes that can see the user in addition to making them go rabid.
+- Red slime core reactions will berserk prometheans that can see the user.
+- Bears will berserk when losing a fight.
+- Changelings can evolve a 2 point ability to use a changeling-specific variant of Berserk, that replaces the text with a 'we' variant.
+Recursive Enhancement allows the changeling to instead used an improved variant that features less exhaustion time and less nutrition drain.
+- Xenoarch artifacts may have forced berserking as one of their effects. This is especially fun if an artifact that makes hostile mobs is nearby.
+Will cause three brainloss in those affected due to the artifact meddling with their mind.
+- A rare alien artifact might be found on the Surface, or obtained from Xenoarch, that causes berserking when it thinks
+the wearer is in danger, however in addition to the usual drawbacks, each use causes three brainloss in the user, due to how
+the artifact triggers the rage.
+
+*/
+
+/datum/modifier/berserk
+	name = "berserk"
+	desc = "You are filled with an overwhelming rage."
+	client_color = "#FF0000" // Make everything red!
+	mob_overlay_state = "berserk"
+
+	on_created_text = "<span class='critical'>You feel an intense and overwhelming rage overtake you as you go berserk!</span>"
+	on_expired_text = "<span class='notice'>The blaze of rage inside you has ran out.</span>"
+	stacks = MODIFIER_STACK_EXTEND
+
+	// The good stuff.
+	slowdown = -1							// Move a bit faster.
+	attack_speed_percent = 0.66				// Attack at 2/3 the normal delay.
+	outgoing_melee_damage_percent = 1.5		// 50% more damage from melee.
+	max_health_percent = 1.5				// More health as a buffer, however the holder might fall into crit after this expires if they're mortally wounded.
+	disable_duration_percent = 0.25			// Disables only last 25% as long.
+	icon_scale_percent = 1.2				// Look scarier.
```

Comment:
Not sure this is desirable, but it won't stop me from merging this.
They shrink back to normal when it ends, yes?

Reply:
Yes.